tests/h/search/index_test.py

```diff
@@ -362,13 +369,15 @@ def test_it_does_not_index_deleted_annotations(self, batch_indexer, es_client, f
                                             id=ann.id)
         assert result_indexed["_id"] == ann.id
 
-        with pytest.raises(elasticsearch1.exceptions.NotFoundError):
+        with pytest.raises(NOT_FOUND_ERROR):
```

Comment:
Ah, I didn't realize `pytest.raise` supported exception type tuples. That's handy.